jinja2/base.html

```diff
@@ -233,11 +236,11 @@
 
     {{ providers_media_js() }}
     {% javascript 'main' %}
-
+    {% if not is_sphinx %}
```

Comment:
Pointed out on IRC that we include analytics.js but not ga.js so removing this makes sense.